src/dopplerUserJson/cmUser.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
What do you think about renaming it in a more descriptive way?

For example: `/src/headerData-for-client-manager-user.json `

Reply:
Yes it sounds more understandable.

Reply:
I've also remembered YAGNI principle.

Reply:
Just curiosity, how does YAGNI apply here?

Reply:
The folder, may not be needed.